Documentation/Plugins/Index.rst

```diff
@@ -1024,6 +1024,60 @@ The fulltext is fetched and rendered by JavaScript into the `<div id="tx-dlf-ful
 
 **Please note**: To allow JavaScript fetching the fulltext, the `CORS headers <https://en.wikipedia.org/wiki/Cross-origin_resource_sharing>`_ must be configured appropriate on the providing webserver.
 
+Model download tool
+^^^^^^^^^^^^^
+
+This tool makes it possible to extract the model URL from the METS file or use the provided model parameter to provide a download URL.
+
+:typoscript:`plugin.tx_dlf_modeldownloadtool.`
+
+.. t3-field-list-table::
+ :header-rows: 1
+
+ - :Property:
+       Property
+   :Data Type:
+       Data type
+   :Values:
+       Values
+
+ - :Property:
```

Comment:
Why do you set up this property here?

`fileGrps...` config is configured in `ext_conf_template.txt` (https://github.com/kitodo/kitodo-presentation/blob/4b3a130ff4794170b0fb4fe1970c3f996309b2df/ext_conf_template.txt#L22)

Please move it there.